ulib/FStar.Math.Lib.fst

```diff
@@ -28,7 +28,10 @@ let slash_star_axiom a b c =
 
 val log_2: x:pos -> Tot nat
 let rec log_2 x =
-  if x >= 2 then 1 + log_2 (x / 2) else 0
```

Comment:
To my knowledge, only one file actually uses the function(examples/crypto/stale/curve_parameters), and that invokes it as `platform_wide - 1 >= log_2 norm_length` , implying that the ceiling is what's actually desired. log_2 is in many other modules such as intlib, defined differently. I only changed Math.Lib.

Changing examples/crypto/stale/curve_parameters would be just as easy, but if preferred maybe there should be log2_n and a c_log2_n for the ceiling.